src/pseudo/NormConservingPsp.jl

```diff
@@ -7,19 +7,21 @@ abstract type NormConservingPsp end
 
 #    lmax::Int                   # Maximal angular momentum in the non-local part
 #    h::Vector{Matrix{Float64}}  # Projector coupling coefficients per AM channel: h[l][i1,i2]
-#    Zion::Int                   # Ionic charge (Z - valence electrons)
 #    identifier::String          # String identifying the PSP
 #    description::String         # Descriptive string
 
 #### Methods:
+# Ionic charge (Z - valence electrons)
+# charge_ionic(psp::NormConservingPsp) = error()
```

Comment:
why are these commented out?

Reply:
Because it's recommended to julia itself error on non-implemented functions rather than explicitly throw an error. I can also completely remove them if you want or make them functions without methods with some docstrings.

Reply:
Yup, thanks!